docs/integrations/data-ingestion/clickpipes/kafka.md

```diff
@@ -33,73 +28,91 @@ You have familiarized yourself with the [ClickPipes intro](./index.md).
-
-<Image img={cp_step4a} alt="Set table, schema, and settings" size="lg" border/>
-
-  You can also customize the advanced settings using the controls provided
-
-<Image img={cp_step4a3} alt="Set advanced controls" size="lg" border/>
+### Configure permissions {#8-configure-permissions}
+ClickPipes will create a dedicated user for writing data into a destination table. You can select a role for this internal user using a custom role or one of the predefined role:
+    - `Full access`: with the full access to the cluster. It might be useful if you use Materialized View or Dictionary with the destination table.
+    - `Only destination table`: with the `INSERT` permissions to the destination table only.
 
-8. Alternatively, you can decide to ingest your data in an existing ClickHouse table. In that case, the UI will allow you to map fields from the source to the ClickHouse fields in the selected destination table.
+<Image img={cp_step5} alt="Permissions" size="md"/>
 
-<Image img={cp_step4b} alt="Use an existing table" size="lg" border/>
+### Complete setup {#9-complete-setup}
+Clicking on "Create ClickPipe" will create and run your ClickPipe. It will now be listed in the Data Sources section.
 
-9. Finally, you can configure permissions for the internal ClickPipes user.
+<Image img={cp_overview} alt="View overview" size="md"/>
 
-  **Permissions:** ClickPipes will create a dedicated user for writing data into a destination table. You can select a role for this internal user using a custom role or one of the predefined role:
-    - `Full access`: with the full access to the cluster. It might be useful if you use Materialized View or Dictionary with the destination table.
-    - `Only destination table`: with the `INSERT` permissions to the destination table only.
+</VerticalStepper>
 
-<Image img={cp_step5} alt="Permissions" size="lg" border/>
+## Schema registries {#schema-registries}
+ClickPipes supports schema registries for Avro data streams.
 
-10. By clicking on "Complete Setup", the system will register you ClickPipe, and you'll be able to see it listed in the summary table.
+### Supported registries {#supported-schema-registries}
+Schema registries that use the Confluent Schema Registry API are supported. This includes:
+- Confluent Kafka and Cloud
+- Redpanda
+- AWS MSK
+- Upstash
 
-<Image img={cp_success} alt="Success notice" size="sm" border/>
+ClickPipes is not currently compatible with the AWS Glue Schema registry or the Azure Schema Registry.
 
-<Image img={cp_remove} alt="Remove notice" size="lg" border/>
+### Configuration {#schema-registry-configuration}
 
-  The summary table provides controls to display sample data from the source or the destination table in ClickHouse
+A schema registry can be configured by when setting up a ClickPipe. This can be configured in one of three ways:
+ 
+1. Providing the root schema registry URL (e.g. `https://registry.example.com`). **This is the preferred method.**
```

Comment:
Actually this is not the preferred method, because it leads to ugly UI.  Without an explicit schema, we can't display it on the next page (because we haven't gotten any data yet.). 

![Screenshot 2025-07-01 at 2 45 48 PM](https://github.com/user-attachments/assets/11068d78-ae7e-4803-9ca9-b4cea2f1b84a)

I would not list it first.  I would think that using the subject is the best, but I don't know what data our customers normally have access to.

Reply:
I would consider reordering this flow to get data first before displaying the schema at some point.